README.md

```diff
@@ -2,3 +2,26 @@
 
 [![Build Status](https://travis-ci.org/dremio-hub/dremio-hbase-connector.svg?branch=master)](https://travis-ci.org/dremio-hub/dremio-hbase-connector)
 
+## Usage Instructions
+
+To add the Dremio Community HBase Connector to the list of Dremio sources:
+
+* From the project root, run `mvn clean install`. 
+The HBase Connector `dremio-hbase-plugin-4.2.2-202004211133290458-b550b6fa.jar` appears in the `target` directory.
```

Comment:
Nit: Consider aligning this text with "From ..." rather than the bullet.

Reply:
@rob, this is a README.md so this alignment will have no effect.